lib/remark-lint-thai.js

```diff
@@ -55,6 +55,13 @@ const rules = [
       if (match[1] !== 'ๆ') return 'Add a space after ๆ'
     },
   },
+  // All about 'กัน' -> กรร
+  {
+    pattern: /กัน(ไกร|เชียง|โชก|ไตร)/g,
```

Comment:
IMO, this can be another spell rule, because `spell` also accepts function arguments.

```suggestion
  spell(/กัน(ไกร|เชียง|โชก|ไตร)/g, m => `กรร${m[1]}`),
```

Reply:
Oh, I never know about this before. Committed!